db/schema.rb

```diff
@@ -10,10 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_04_26_011733) do
+ActiveRecord::Schema[7.0].define(version: 2023_04_27_204831) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "pair_requests", force: :cascade do |t|
```

Comment:
looking at this, the migration seems off. check the format for adding `foreign_key` to a migration.